exercises/protein-translation/protein_translation_test.go

```diff
@@ -59,3 +59,15 @@ func TestProtein(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkCodon(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		FromCodon(codonTestCases[0].input)
```

Comment:
This is a good point, and usually what we can do is create sub-benchmarks that would bomb out in case one earlier route took longer than expected.

https://blog.golang.org/subtests

Can refactor and resubmit. In all honesty we should do this across the other challenges too but maybe will save that for if/when we can become maintainers :D

Reply:
Most benchmarks use an inner for loop over the test cases.  I didn't notice that omission on my first look. Good catch @hilary .